rele/subscription.py

```diff
@@ -3,20 +3,29 @@
 import time
 
 from django import db
-from django.conf import settings
 
 logger = logging.getLogger(__name__)
 
 
 class Subscription:
 
-    def __init__(self, func, topic, suffix=None):
+    def __init__(self, func, topic, prefix=None, suffix=None):
```

Comment:
Could this be 
```suggestion
    def __init__(self, func, topic, prefix='', suffix=''):
```
instead? And then down below you don't have to do an `or`

Reply:
5aa98ab